### DISCUSSION TITLE
"Failed to send shoutrrr notification" error="error getting SMTP client: error connecting to server: dial tcp: lookup smtp.gmail.com on 127.0.0.11:53: server misbehaving"

### DISCUSSION THREAD

**Discussion by antgustech** [containrrr/watchtower#1853] [Q&A]
I recently added adguardhome to my home network to block ads. My router is configured to use it so that all devices get to use it as DNS.

The strange thing now that since I switched DNS, watchtower email notificaiton no longer works. When I start the container, I get 

`"Failed to send shoutrrr notification" error="error getting SMTP client: error connecting to server: dial tcp: lookup smtp.gmail.com on 127.0.0.11:53: server misbehaving"`

I can see in the adguard dashboard that the google server is allowed through though. And I can ping it and do nslookup.

![error](https://github.com/containrrr/watchtower/assets/8721113/6276e006-f97a-4e10-87ac-ff9b1f4d0046)

Anyone else experienced similar issue? Not quite sure why it checks 127.0.0.11:53: since that is not my DNS server that I can see in /etc/resolv/conf.

I have also tried setting up Discord notification and I get the same error:

`dial tcp: lookup discord.com on 127.0.0.11:53: server misbehaving" index=0 notify=no service=discord`

**Comment by piksel** [comment]
The 127.0.0.11 is the docker DNS that it uses to resolve the names of other containers. Inside the docker containers, that is the configured DNS server.

**Reply by antgustech** [reply]
Yes but I guess that is somehow redirected to the host machine DNS? Because I do see the request coming through.

**Reply by piksel** [reply]
Yeah, it should propagate to your local DNS. The error means that the application gets a SERVFAIL response from the DNS. I found some older issues related to this, but not sure if they are related to your environment. 
Could you post your resolv.conf (you can replace the IPs if you want)?

Also, if you just run a plain `ubuntu:latest` (or similar) container and run `nslookup discord.com`, does that give you a valid response?

**Comment by antgustech** [comment]
> Yeah, it should propagate to your local DNS. The error means that the application gets a SERVFAIL response from the DNS. I found some older issues related to this, but not sure if they are related to your environment. Could you post your resolv.conf (you can replace the IPs if you want)?
> 
> Also, if you just run a plain `ubuntu:latest` (or similar) container and run `nslookup discord.com`, does that give you a valid response?


Sorry for the late reply, I havent had time to check this.

I reverted from using my own custom DNS to simply use Cloudflared. So now my resolv.conf has their server:

nameserver 1.1.1.1

And mind all other containers I have running seem to work perfectly fine.

I added Ubuntu and nslookup discord.com gives me:

```
nslookup discord.com
Server:         1.1.1.1
Address:        1.1.1.1#53

Non-authoritative answer:
Name:   discord.com
Address: 162.159.136.232
Name:   discord.com
Address: 162.159.128.233
Name:   discord.com
Address: 162.159.137.232
Name:   discord.com
Address: 162.159.135.232
Name:   discord.com
Address: 162.159.138.232
```

And things seem to work just fine there,  i could install packages, curl google.com etc.

**Comment by antgustech** [comment]
Any updates on this? :)
